Add disc theta band

viniwiedemann opened the pull request:
adds the disc theta band off region

viniwiedemann committed:
Create theta band off region 

Code for the theta band disc off region mask to compensate for the declination dependance

viniwiedemann committed:
Merge pull request #1 from viniwiedemann/viniwiedemann-patch-1

Create theta band off region

viniwiedemann committed:
Delete disc_theta_band_off_region.py

viniwiedemann committed:
Add disc theta band off region

Added disc-theta-band-off-region to compensate for the declination dependance

viniwiedemann committed:
Merge pull request #2 from viniwiedemann/viniwiedemann-patch-2

Add disc theta band off region

jrbourbeau reviewed (commented):
Thanks for the PR @viniwiedemann, `disc_theta_band_off_region` will be a nice addition! I have a few comments

1. It looks like there are some code style issues that are causing the CI to fail https://travis-ci.org/jrbourbeau/sky-anisotropy/jobs/408712288#L828-L853

2. Could you add `disc_theta_band_off_region` to the `off_region_funcs` dictionary in `core.py`? That way it can be used in `on_off_chi_squared_single`

3. Could you add `disc_theta_band_off_region` to the `sky_anisotropy/__init__.py` file with the other off region functions

Please let me know if you have any questions about the above comments

viniwiedemann committed:
codestyle

fixing some issues with the codestyle and adding disc theta band to the off region fct dictionary

viniwiedemann committed:
Update core.py

viniwiedemann committed:
adding theta_band_off_region
